apps/backend/src/mcp/tools/container_management.py

```
@@ -34,112 +32,108 @@ async def list_containers(
-            
+
             for container_data in container_data_list:
                 # Extract basic container information
                 config = container_data.get("Config", {})
                 state = container_data.get("State", {})
                 network_settings = container_data.get("NetworkSettings", {})
-                
+
                 # Extract Docker Compose information from labels
                 labels = config.get("Labels") or {}
                 compose_project = labels.get("com.docker.compose.project", "")
```

Ruff (0.12.2)

Local variable `compose_project` is assigned to but never used

Remove assignment to unused variable `compose_project`

(F841)